cypress/e2e/shared/flowsSidebarExportToClipboard.test.ts

```diff
@@ -4,39 +4,48 @@ const openCopyAs = () => {
   cy.getByTestID('sidebar-button')
     .first()
     .scrollIntoView()
-    .click({force: true})
+    .click()
   cy.getByTestID('Export to Client Library--list-item').click()
 }
 
 const addFluxQueryInNotebook = (query: string) => {
   cy.getByTestID('add-flow-btn--rawFluxEditor').click()
-  cy.getByTestID('flux-editor').monacoType(`{selectall}{del}${query}`)
+  cy.wait('@NotebooksPatchRequest')
+  cy.getByTestID('flux-editor').should('be.visible')
+
+  /* Due to the way the default text populates
+   *   - do not use .monacoType
+   *   - must select the first visible line
+   */
+  cy.get('.monaco-editor .view-line:first').should('be.visible')
+  cy.get('.monaco-editor .view-line:first').type(
+    `{downarrow}{downarrow}${query}`
+  )
 }
 
 const createEmptyNotebook = () => {
-  cy.intercept('PATCH', `/api/v2private/notebooks/*`, req => {
-    req.alias = 'NotebooksPatchRequest'
-  })
-
```

Comment:
Intercept in the `beforceEach` and wait on the `PATCH` request 5 times. Yes, they happen 5 times per test and should be caught each time.